backend/cabin-sorting/tsconfig.json

```diff
@@ -1,109 +1,46 @@
 {
   "compilerOptions": {
```

Comment:
is there a reason to include ALL of these settings?

Reply:
I believe I transferred the options over from judging algo but it does work without all of the included options so i reverted it to what I had